### ISSUE TITLE
FluentAssertions.Analyzers 0.19.0 causes compile errors in .NET 7 projects

### ISSUE THREAD

**Issue by RaceProUK** [fluentassertions/fluentassertions.analyzers#188]
### Description

FluentAssertions.Analyzers 0.19.0 causes compile errors in .NET 7 projects

### Complete minimal example reproducing the issue

A test project targetting .NET 7 with FluentAssertions.Analyzers 0.19.0 added via NuGet

### Expected behavior:

Test project compilation succeeds

### Actual behavior:

Test project compilation fails with errors similar to the following:
An instance of analyzer FluentAssertions.Analyzers.Xunit.AssertTrueAnalyzer cannot be created from C:\Users\[redacted]\.nuget\packages\fluentassertions.analyzers\0.19.0\analyzers\dotnet\cs\FluentAssertions.Analyzers.dll : Could not load file or assembly 'System.Runtime, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.

### Versions

* Which version of Fluent Assertions Analyzers are you using? 0.19.0
* Which .NET runtime and version are you targeting? .NET 7

**Comment by KuraiAndras** [comment]
Yes, we just hit the same issue. I personally think that keeping the project on .NET standard 2.0 would be beneficial, or target the same platforms as the main FluentAssertions package (core 2.1, core 3.0, .NET Framework 4.7, .NET Standard 2.0, .NET Standard 2.0, .NET 6). Even the [microsoft docs](https://learn.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/tutorials/how-to-write-csharp-analyzer-code-fix#explore-the-analyzer-template) recommends using .NET Standard 2.0

**Comment by RaceProUK** [comment]
That would work even better, as then no change would be needed for the upcoming .NET 8
